sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1128,16 +1128,16 @@ private static string CreateContinuationTokenFromHeaders(TableQueryEntitiesHeade
             }
         }
 
-        private static (string NextPartitionKey, string NextRowKey) ParseContinuationToken(string continuationToken)
+        internal static (string NextPartitionKey, string NextRowKey) ParseContinuationToken(string continuationToken)
         {
             // There were no headers passed and the continuation token contains just the space delimiter
-            if (continuationToken?.Length <= 1)
+            if (continuationToken is null || continuationToken.Length <= 1)
             {
                 return (null, null);
             }
 
-            var tokens = continuationToken.Split(' ');
-            return (tokens[0], tokens.Length > 1 ? tokens[1] : null);
+            var tokens = continuationToken.Split(new[] { ' ' }, 2);
```

Comment:
should we make the char array static to avoid allocating for each response?

Reply:
Yeah, great idea! Done.